enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -58,7 +61,7 @@ export function useLoadQuestion({
 
   // Keep track of the latest question and query results.
   // They can be updated from the below actions.
-  const [questionState, setQuestionState] = useReducer(questionReducer, {});
+  const [questionState, mergeQuestionState] = useReducer(questionReducer, {});
```

Comment:
This is also a rename, just to make it clearer. Normally, I'd expect `setSomething` to override the whole thing. Let me know if you think otherwise, I can revert this if it makes more sense for you guys.

Reply:
Agree!